### ISSUE TITLE
Automate updates

### ISSUE THREAD

**Issue by nsolomon-nycc** [NewYorkCityCouncil/PA_evictions#1]
@romartinez-nycc Do you still need my help getting automatic updates set up?

**Comment by romartinez-nycc** [comment]
Yes! I won't have time today to work on this. But, Monday I will.

**Comment by romartinez-nycc** [comment]
@nsolomon-nycc Got the script running well. Just need some help to test the cronjob :)
